tests/dummy/app/index.html

```diff
@@ -14,6 +14,16 @@
     <link rel="stylesheet" href="assets/vendor.css">
     <link rel="stylesheet" href="assets/dummy.css">
 
+    <script type="text/javascript"
+        src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.6.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+    </script>
+    <script type="text/x-mathjax-config">
```

Comment:
@abought maybe there was a reason MathJax was being installed this way? https://github.com/CenterForOpenScience/ember-preprints/pull/129/files#diff-b9707fe88797285e93483456d1389b61